modules/Messenger/messenger_postProcess.php

```diff
@@ -1720,11 +1720,11 @@
                                       $resultSMS->execute($dataSMS);
                                     }
                                     catch(PDOException $e) { }
-                                    while ($rowSMS=$resultSMS->fetch()) {
+                                    while ($rowEmail=$resultSMS->fetch()) {
```

Comment:
Thanks for your PR! Rather than changing this variable to $rowEmail, it may make sense in the code to change the following instances to $rowSMS instead, to keep things consistent.

```suggestion
                                    while ($rowSMS=$resultSMS->fetch()) {
```